### DISCUSSION TITLE
Suddenly, CoPilot stopped suggesting

### DISCUSSION THREAD

**Discussion by ghost** [community/community#33540] [Copilot Conversations]
Hi,
Suddenly, my Copilot stopped suggesting inline code in my Jupyter notebook.  I tried something simple like 
` # print hello`
... and nothing.
I created a test.py file and it works OK over there. 
But I wonder if there's a conflict between the Jupyter and Github Copilot AI extensions in VS Code.
Any suggestions on how to turn it on again?  Here's what I tried so far:
1) exited and restarted VSCode
2) disabled and re-enabled GitHub Copilot AI
3) tried logging into GitHub from within VSCode
4) tried uninstalling and reinstalling GitHub Copilot AI
5) tried restarting my computer
Still no luck...   other ideas?

Yours,

Raj

**Comment by ghost** [comment]
Update:  I notice that it stops working after I input the following code.


`x = df['Order Line Trans Calendar Month'].str.extract(pattern)`
`# print hello`
... nope.


but magically, it continues to work after this line

`df['Calendar Month'].str.extract(r'(\d\d\d\d)-M(\d\d)')`

**Comment by iamcjwilliams** [comment]
@rajsodhi Have you updated to the latest version?
